portfolio.css

```diff
@@ -0,0 +1,346 @@
+@import url("https://fonts.googleapis.com/css2?family=Poppins:wght@400;500;600;700;800;900&display=swap");
+
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+body {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  background: #e5e5e5;
+  font-family: "Poppins", sans-serif;
+}
+
+.first {
+  width: 375px;
+  background: #f7f7f9;
+}
+
+header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  height: 48px;
+}
+
+.bg-white {
+  background-color: #fff;
+}
+
+.logo {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  padding: 10px 12px;
+  margin-left: 12px;
+  margin-top: 4px;
+}
+
+.logo a {
+  text-decoration: none;
+  font-weight: 700;
+  font-size: 18px;
+  line-height: 20px;
+  color: #6070ff;
+}
+
+nav {
+  padding: 8px;
+}
+
+.menu {
+  width: 24px;
+  height: 24px;
+  margin-right: 12px;
+  cursor: pointer;
+}
+
+section {
+  display: grid;
+  grid-template-columns: 1fr;
+  padding: 114px 24px;
+  gap: 12px;
+}
+
+ul {
+  list-style: none;
+}
+
+.headline {
+  background-image: url("images/bg-image.png");
+  border-bottom-left-radius: 60px;
+}
+
+.title {
+  font-weight: 700;
+  font-size: 40px;
+  line-height: 52px;
+  color: #172b4d;
+}
+
+.details {
+  font-weight: 400;
+  font-size: 16px;
+  line-height: 24px;
+  color: #344563;
+}
+
+.contact-link {
+  text-decoration: none;
+  font-weight: 500;
+  font-size: 16px;
+  line-height: 24px;
+  color: #7f8cff;
+}
+
+.social-media {
+  display: flex;
+  flex-direction: column;
+  gap: 16px;
+}
+
+.social-media-links {
+  display: flex;
+  gap: 20px;
+  text-decoration: none;
+}
+
+.entire {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  padding: 114px 24px;
+  gap: 88px;
+}
```

Comment:
- Great Job for using a grid layout  in different places, I think you have to use here a grid to lay out your work section container as it is part of the [requirement](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/2_mobile.md) 👍 <img width="752" alt="Screenshot 2023-06-06 105306" src="https://github.com/jerymey/My-portfolio/assets/52010789/53dcb252-d547-4707-a6b9-3290144c9c42">